.github/workflows/ci.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Select Xcode Version
         run: sudo xcode-select --switch /Applications/Xcode_16.app/Contents/Developer
       - name: Lint Podspec
-        run: bundle exec pod lib lint --verbose --fail-fast --swift-version=6.0
+        run: bundle exec pod lib lint --verbose --fail-fast --swift-version=6.0 --allow-warnings # Cocoapods v1.6 now warns about potential naming colisions. We can fix this in the next breaking change.
```

Comment:
The warnings:
```
    - WARN  | xcodebuild:  /Users/runner/work/Valet/Valet/Sources/Valet/Accessibility.swift:21:13: warning: public enum 'Valet.Accessibility' shadows module 'Accessibility', which may cause failures when importing 'Valet' or its clients in some configurations; please rename either the enum 'Valet.Accessibility' or the module 'Accessibility', or see https://github.com/apple/swift/issues/56573 for workarounds
    - WARN  | xcodebuild:  /Users/runner/work/Valet/Valet/Sources/Valet/Valet.swift:22:20: warning: public class 'Valet.Valet' shadows module 'Valet', which may cause failures when importing 'Valet' or its clients in some configurations; please rename either the class 'Valet.Valet' or the module 'Valet', or see https://github.com/apple/swift/issues/56573 for workarounds
```

This is indeed a thing. I doubt we'll fix `Valet.Valet`, but `Valet.Accessibility` could be renamed pretty easily in the next breaking change.